src/sagemaker_training/mpi.py

```diff
@@ -95,11 +103,34 @@ def run(
 
         if wait:
             logger.info("Waiting for MPI process to finish.")
-            _wait_orted_process_to_finish()
+            gone, alive = _wait_orted_process_to_finish()
+            logger.info(f"Reporting status for ORTEd process. gone: {gone} alive: {alive}")
             logger.info("Orted process exited")
             time.sleep(30)
+            logger.info(f"Begin looking for status file on {self._current_host}")
+            status_file = MPI_FINISHED_STATUS_FILE + "." + self._master_hostname
+            file_found = self._wait_for_status_file(status_file)
+            if file_found:
+                logger.info("Status file found. Exit gracefully")
```

Comment:
maybe `MPI training job status file found` here to make the print more informative?

Reply:
updated the print statement.